agents/sorenson_dlm/pfeiffer_tpg366_agent.py

```diff
@@ -0,0 +1,53 @@
+#script to log and readout pfeiffer TPG 366 gauge contoller
```

Comment:
This seems like a duplicate of an old copy of the pfeiffer Agent, perhaps meant as a starting point for the DLM agent? If nothing has been done to this to start the DLM agent, please remove it from this PR. The DLM agent should be a separate PR.